src/mixins/bem/index.js

```diff
@@ -1,14 +1,11 @@
-import { mapFacets, createBemClass, DEFAULT_OPTIONS } from './helpers';
-
-const PROPS_CONFIG = {
-    facets: {
-        type: Array,
-        default() {
-            return [];
-        },
-    },
-};
+import { getComputedConfig, getPropsConfig, mapFacets, createBemClass, DEFAULT_OPTIONS } from './helpers';
 
+/**
+ * Add BEM related helpers to a component
+ * @param {string} bemRoot - The block name
+ * @param {IBemMixinOptions} config - The mixin config
```

Comment:
AFAIK config is an optional/nullable parameter

Reply:
Fixed